src/components/Library.tsx

```diff
@@ -72,7 +73,7 @@ export const Library = ({ library }: Props) => {
         )}
       </div>
       <button id="backToTopBtn" onClick={backToTop}>
-        <span className="material-icons">arrow_drop_up</span>
+        <ArrowDropUp className="material-icons"></ArrowDropUp>
```

Comment:
And this one you can simple use:
`<ArrowDropUp className="material-icons" />`
😄